python/ray/serve/BUILD

```diff
@@ -549,4 +549,12 @@ py_test(
     srcs = serve_tests_srcs,
     tags = ["exclusive", "team:serve"],
     deps = [":serve_lib"],
+)
+
+py_test(
+    name = "test_common",
+    size = "small",
+    srcs = serve_tests_srcs,
+    tags = ["exclusive", "team:serve"],
+    deps = [":serve_lib"],
```

Comment:
This test file wasn't running in CI before? 😮

Reply:
exactly...